master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/ha/HAMasterMetaManager.java

```diff
@@ -278,7 +278,7 @@ public void handleRegisterWorker(
   }
 
   @Override
-  public void handleReportWorkerFailure(List<WorkerInfo> failedNodes, String requestId) {
+  public void handleReportWorkerUnavailable(List<WorkerInfo> failedNodes, String requestId) {
```

Comment:
Should also rename Type.ReportWorkerFailure

Reply:
Done